src/components/Modals/NewMatchConfirmationModal.tsx

```diff
@@ -82,46 +124,66 @@ const NewMatchConfirmationModal: React.FC<{
                 überprüfen.
               </p>
 
-              <strong>
-                <p>
-                  {user.firstname} {user.lastname}
-                </p>
-              </strong>
+              {/* <strong> */}
```

Comment:
Instead of commenting these lines out you can also delete them. In case we are willing to restore it, we can simply rollback using git. For now, it would make the code cleaner 😉